packages/devtools_app/lib/src/inspector/flutter/story_of_your_layout/arrow.dart

```diff
@@ -37,6 +69,11 @@ class BidirectionalArrowWrapper extends StatelessWidget {
         distanceToArrow = direction == Axis.horizontal
             ? EdgeInsets.symmetric(horizontal: distanceToArrow)
             : EdgeInsets.symmetric(vertical: distanceToArrow),
+        isBidirectional = true,
+        firstArrowType =
```

Comment:
nit: instead of firstArrowType and secondArrowType how about
startArrowType and endArrowType?

Reply:
that's a very good idea!